public/content/security/index.md

```diff
@@ -125,15 +125,15 @@ One of the biggest reasons people get scammed in crypto generally is a lack of u
 
 **Never, for any reason, share your private keys!**
 
-The private key to your wallet acts as a password to your Ethereum wallet. It is the only thing stopping someone who knows your wallet address from draining your account of all of its assets!
+The private key to your wallet is a password to your Ethereum wallet. It is the only thing stopping someone who knows your wallet address from draining your account of all of its assets!
```

Comment:
My side-concern with this statement is it can give people the false impression that they need to enter this "password" when they use their account... vs a password used to encrypt/decrypt your wallet, which works differently and is more commonly the "password" users will need to actually remember or have easy access to (unlike their private key which they may never see)

I see that as out-of-scope to the current changes though, just noting for the future